bindings/python/test_x86.py

```diff
@@ -47,10 +47,19 @@ def print_string_hex(comment, str):
 
     # print modRM byte
     print("\tmodrm: 0x%x" % (insn.modrm))
-
+	
+	# print modRM offset
```

Comment:
please fix indentation here, with spaces in Python code (this is regression i think)

Reply:
and print these fields only when they are relevant.